packages/superset-ui-core/src/query/types/PostProcessing.ts

```diff
@@ -0,0 +1,118 @@
+import { JsonObject } from '../../connection';
+import { TimeGranularity } from '../../time-format';
+
+/**
```

Comment:
it should be before all imports

Reply:
Right. Thanks for catching this!  The imported were added by VS Code autocomplete and I forgot to check.